examples/apollo-federation/gateway/gateway.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
I think we can still lint examples, to avoid future errors, some things we can explicitly ignores like `eslint-disable-next-line` or `eslint-disable-line` but not full file

Reply:
these `node` examples it was complaining a lot about `require` not defined and do not use `require` so I decide to disable linting on full example

Reply:
Why not replace require with import? Since node 12 is deprecated

Reply:
Wouldn’t we need to be on node 16+ for that?

Reply:
Node 16 is good!